apiv2/project/project.go

```diff
@@ -593,6 +598,82 @@ func (c *RESTClient) DeleteProjectRobot(ctx context.Context, p *modelv2.Project,
 	return nil
 }
 
+// ListProjectWebhookPolicies returns a list of all webhook policies in project p.
+func (c *RESTClient) ListProjectWebhookPolicies(ctx context.Context, p *modelv2.Project) ([]*model.WebhookPolicy, error) {
+	if p == nil {
+		return nil, &ErrProjectNotProvided{}
+	}
+
+	resp, err := c.LegacyClient.Products.GetProjectsProjectIDWebhookPolicies(
+		&products.GetProjectsProjectIDWebhookPoliciesParams{
+			ProjectID: int64(p.ProjectID),
+			Context:   ctx,
+		}, c.AuthInfo)
+	if err != nil {
+		return nil, handleSwaggerProjectErrors(err)
+	}
+
+	return resp.Payload, nil
+}
+
+// AddProjectWebhookPolicy adds a webhook policy to project p.
+func (c *RESTClient) AddProjectWebhookPolicy(ctx context.Context, p *modelv2.Project, policy *model.WebhookPolicy) error {
+	if p == nil {
+		return &ErrProjectNotProvided{}
+	}
+
+	if policy == nil {
+		return &ErrProjectNoWebhookPolicyProvided{}
+	}
+
+	_, err := c.LegacyClient.Products.PostProjectsProjectIDWebhookPolicies(
+		&products.PostProjectsProjectIDWebhookPoliciesParams{
+			Policy:    policy,
+			ProjectID: int64(p.ProjectID),
+			Context:   ctx,
+		}, c.AuthInfo)
+	return handleSwaggerProjectErrors(err)
+}
+
+// UpdateProjectWebhookPolicy updates a webhook policy in project p.
+func (c *RESTClient) UpdateProjectWebhookPolicy(ctx context.Context, p *modelv2.Project, policyID int, policy *model.WebhookPolicy) error {
+	if p == nil {
+		return &ErrProjectNotProvided{}
+	}
+
```

Comment:
We should check on the WebhookPolicy object being nil here, too.
```suggestion
	if policy == nil {
		return &ErrProjectNoWebhookPolicyProvided{}
	}
```